tests/test_events/test_api.py

```diff
@@ -260,3 +276,16 @@ def decorated_function() -> None:
         assert not EventAPI().sent_events
         assert not EventAPI().pending_events
         assert not EventAPI().failed_events
+
+    def test_clears_que_when_wrapping_without_calling(self) -> None:
```

Comment:
You have typo here but also it would be good to describe what is expected:
```suggestion
    def test_always_clears_queues_before_actually_decorating_function(self) -> None:
```